alert-common/src/main/java/com/synopsys/integration/alert/common/enumeration/UserRole.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * alert-common
+ *
+ * Copyright (c) 2019 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.common.enumeration;
+
+public enum UserRole {
+    ALERT_ADMIN,
+    ALERT_JOB_MANAGER,
+    ALERT_USER;
+
+    public static final String ALERT_ADMIN_TEXT = "ALERT_ADMIN";
```

Comment:
Why do we need these Strings? Are these not the same as the enum.toString()?

Reply:
In the integration tests the @WithMockUser annotation must use a constant.  I can remove these string literals in here and refactor the tests to reference a constant class just for the tests to clean this up.